internal/metamorph/processor.go

```diff
@@ -210,24 +216,29 @@ func (p *Processor) StartProcessMinedCallbacks() {
 
 func (p *Processor) StartProcessSubmittedTxs() {
 	p.waitGroup.Add(1)
-
+	ticker := time.NewTicker(p.bulkProcessInterval)
 	go func() {
 		defer p.waitGroup.Done()
+
+		var reqs []*ProcessorRequest
```

Comment:
I suggest to use a slice with some capacity declared already to avoid multiple reallocation
`reqs:= make([]*ProcessorRequest, 0, 100)`